assets/src/design-system/components/checkbox/index.js

```diff
@@ -37,6 +37,7 @@ const Border = styled.div(
     width: ${CONTAINER_WIDTH}px;
     border-radius: ${theme.borders.radius.small};
     border: ${BORDER_WIDTH}px solid ${theme.colors.border.defaultNormal};
+    pointer-events: none;
```

Comment:
if we need to revert to this that's fine, but we should have a way for the cursor to still be a pointer for checkboxes.